config/mainCfg_MuMu_UL18.cfg

```diff
@@ -1,52 +1,49 @@
 [general]
-
 lumi = 59741 # pb^-1
 lumi_fb = 60.0 # fb^-1
 
-data = DsingleMuA, DsingleMuB, DsingleMuC, DsingleMuD
+data = DsingleEleA, DsingleEleB, DsingleEleC, DsingleEleD, DMETA, DMETB, DMETC, DMETD
```

Comment:
I think these should probably stay SingleMuon datasets 😄

Reply:
Thank you for spotting this.